.gitignore

```diff
@@ -8,13 +8,20 @@ __pycache__/
 *.pt
 .ftpconfig
 .idea/
+.vscode/
 
-examples/data/cifar*
 data/plenoptic-test-files
 data/ssim_images
 data/ssim_analysis.mat
 data/cat7*
 data/elep*
+data/cifar*
+
+logs/
+
+train
+
+examples/geodesics_development.ipynb
```

Comment:
Same question about this -- I think `.gitignore` should basically include things that we can expect others to have (hence all the `data/` things that are created by our code or downloading models from torchvision). I think `examples/geodesics_development.ipynb` doesn't belong here for that reason (since it's a specific notebook), and I'm not sure about `logs/` and `train`. How do they get created?